app/src/main/java/me/quadphase/qpdex/PokedexActivity.java

```diff
@@ -40,20 +111,35 @@ public void onClick(View view) {
         });
         }
 
+
+        //This is a test, remove before pushing to master
```

Comment:
I'll probably keep it until I can actually fill the list with non-placeholders...